sunxi: Fix WiFi on Sinovoip Banana Pi M2 Ultra

halmartin opened the pull request:
This enables onboard WiFi (AP6212) out of the box

Suggestion from @juanesf. Tested on trunk, onboard WiFi is now working.

Signed-off-by: Hal Martin <hal.martin@gmail.com>

adrianschmutzler commented:
Please separate name change and package changes into two separate commits (inside this PR).

adrianschmutzler commented on target/linux/sunxi/image/cortex-a7.mk:
```diff
@@ -63,13 +63,14 @@ define Device/lemaker_bananapi
+  DEVICE_VENDOR := Sinovoip
   DEVICE_MODEL := Banana Pi M2 Ultra
-  DEVICE_PACKAGES:=kmod-rtc-sunxi kmod-ata-sunxi
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-ata-sunxi kmod-brcmfmac \
```
Please don't introduce spaces next to the equal sign, as this does not match the style of the remaining entries.

adrianschmutzler commented on target/linux/sunxi/image/cortex-a7.mk:
```diff
@@ -63,13 +63,14 @@ define Device/lemaker_bananapi
   DEVICE_MODEL := Banana Pi M2 Ultra
-  DEVICE_PACKAGES:=kmod-rtc-sunxi kmod-ata-sunxi
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-ata-sunxi kmod-brcmfmac \
+	 brcmfmac-firmware-43430a0-sdio wpad-basic
```
Indent with one tab, no space.

adrianschmutzler commented:
If this target uses sysupgrade, the name change will also require SUPPORTED_DEVICES to be extended.

halmartin commented:
> If this target uses sysupgrade, the name change will also require SUPPORTED_DEVICES to be extended.

Only sdcard images are available, so I don't think this applies?

adrianschmutzler commented:
Your commits have formal issues concerning their commit titles (prefix, length).
Despite, please add a justification/evidence for your second commit.
In the first commit, you should leave the vendor name as it is.

halmartin committed:
sunxi: BPi-M2U kernel modules for onboard WiFi

Signed-off-by: Hal Martin <hal.martin@gmail.com>

halmartin committed:
sunxi: Correct manufacturer name to Sinovoip

Sinovoip is the manufacturer of the BPi-M2U:
http://www.banana-pi.org/m2u.html

Signed-off-by: Hal Martin <hal.martin@gmail.com>

halmartin commented:
I made an error when the device was added to OpenWrt, LeMaker was the manufacturer of the original Banana Pi but the Banana Pi M2 Ultra is manufactured by Sinovoip:
http://www.banana-pi.org/m2u.html (see page footer)

I have amended the commits as requested.

hauke commented:
Thank you for the patches, they were applied to master.
